src/scripts/tray.js

```diff
@@ -87,23 +98,63 @@ function createAppTray () {
     };
 }
 
+function setTitle (title) {
```

Comment:
I did not know how to call it. I decided to call it setTitle, because it does what the function on OSx does - set a picture with some title. In our situation it is mostly a number for pictures. I can change the name to setImage or something like this.